### PR TITLE
Adding Instabug Integration

### PR DESCRIPTION
**Pull request by Ahmedanwer**

### CODE REVIEW

File: integrations.yml [hunk 1]
```diff
@@ -257,3 +257,9 @@
   website: "https://enjoyfieldtrip.com/?utm_source=basecamp&utm_medium=extras"
   icon: "fieldtrip.png"
   category: 1
+-
+  name: "Instabug"
+  description: "Integrating Instabug with Basecamp helps your team receive all the details they need about issues reported by testers and users from your mobile app using our SDK. With this integration, all bugs and user feedback can be converted into issues with just a click!"
```
Comment (jeremy) [inline integrations.yml]: Could you clarify this description and edit it down to a short phrase that pitches what Instabug does for Basecamp users? Use Basecamp terminology, too: we have to-dos, not issues.

**Comment by Ahmedanwer** [comment]
@jeremy can you please check again

File: integrations.yml [hunk 2]
```diff
@@ -257,3 +257,9 @@
   website: "https://enjoyfieldtrip.com/?utm_source=basecamp&utm_medium=extras"
   icon: "fieldtrip.png"
   category: 1
+-
+  name: "Instabug"
+  description: "Receive detailed bug reports from your mobile app directly to Bascamp to-dos with instabug."
```
Comment (jeremy) [inline integrations.yml]: Bascamp → Basecamp
instabug → Instabug

**Comment by Ahmedanwer** [comment]
@jeremy can you check again

**Pull request merged**